lib/skroutz/sku.rb

```diff
@@ -1 +1,8 @@
-class Skroutz::Sku < Skroutz::Resource; end
+class Skroutz::Sku < Skroutz::Resource
+  association :category
+  association :similar, class_name: 'Sku'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.